marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx

```diff
@@ -151,6 +151,46 @@ describe('Create/Edit Mapping Step artifact component', () => {
 
   });
 
+  test('Verify clicking "Save" with neither name nor collection shows error', () => {
+    const { getByText, getByLabelText, getByPlaceholderText } = render(<CreateEditMappingDialog {...data.newMap} />);
+
+    fireEvent.click(getByText('Save'));
+
+    // both messages should show when both boxes are empty
+    expect(getByText('Name is required')).toBeInTheDocument(); 
+    expect(getByText('Collection or Query is required')).toBeInTheDocument();
+  });
+
+  test('Verify clicking "Save" with name but no collection shows error', () => {
```

Comment:
Same as my earlier comment. We can club the tests for errors in name and collection field together here as well, to avoid re-renders.

Reply:
fixed